docs/plugins/fetchart.rst

```diff
@@ -173,6 +173,14 @@ default engine searches the entire web for cover art.
 Note that the Google custom search API is limited to 100 queries per day.
 After that, the fetchart plugin will fall back on other declared data sources.
 
+Cover art archive
+''''''''''''''''''''
+
+The coverart album art source fetch cover art for a specific release using the
+musicbrainz release id. To fetch cover art for the release group, use
+the coverartreleasegroup source which fetch cover art using the musicbrainz
+release group id.
```

Comment:
Here, it would be nice to link to http://coverartarchive.org for more information about the source.

Also, here's an idea for alternative wording:

```
The `coverart` source fetches images from the `Cover Art Archive`_. The
default source prioritizes the MusicBrainz release ID: i.e., you get release-
specific art if it's available. To instead use only *release group* album art,
use the source named `coverartreleasegroup`. This will, for example, avoid
using the album art for a specific special edition of an album.

.. _Cover Art Archive: http://coverartarchive.org
```

Reply:
Also, can you please mention the new name of the source in the "Configuration" section, where we list all the available sources?